### PR TITLE
Math magicians: fetch data from API

### PR DESCRIPTION
**Pull request by arnoldnekemiah**
I did the following
- Used the following external API: [API Ninjas](https://api-ninjas.com/api/quotes)
- Created a new component for displaying a quote
- Fetched data inside of a new component
- Showed a loading state and error state
- Imported the new component into your main component (next to the Calculator)

image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -42,3 +42,16 @@ button {
 .equal {
   background-color: orange;
 }
+
+.quote-container {
+  /* margin-top: 50px; */
```
Comment (anagudelogu) [inline src/App.css]: - [x] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧

File: src/components/apiUse.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      const headers = {
+        'X-Api-Key': 'aDqrgH1nQfLU2aLHs5rk4g==b0VPzAZwpMAqqzyO',
+      };
+
+      const response = await fetch(
+        'https://api.api-ninjas.com/v1/quotes?category=success',
+        { headers },
+      );
+
+      if (response.status >= 200 && response.status <= 299) {
```
Comment (anagudelogu) [inline src/components/apiUse.js:21]: - [x] **[OPTIONAL]** You could replace this with: 

```suggestion
      if (response.ok) {
```

According to [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok):

>The ok read-only property of the [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) interface contains a Boolean stating whether the response was successful (status in the range 200-299) or not.

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @arnoldnekemiah, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @arnoldnekemiah, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**